### PR TITLE
Enabled Fast Block Propagation by default

### PR DESCRIPTION
**Pull request by rmoreliovlabs**
We wan to enable fast block proagation by default

## Description
Updated all `.conf` to enable this feature by default.

## Motivation and Context
We wan to enable fast block proagation by default because it's faster to validate and process the blocks. This behavior has been tested here: https://rsklabs.atlassian.net/browse/CORE-567

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Requires Activation Code (Hard Fork)

* **Other information**:

### CODE REVIEW

File: rskj-core/src/main/resources/reference.conf [hunk 1]
```diff
@@ -121,7 +121,7 @@ peer {
     }
 
     # flag that allows to propagate a received block without executing it and only checking basic validation rules.
-    fastBlockPropagation = false
+    fastBlockPropagation = true
```
Comment (Vovchyk) [inline rskj-core/src/main/resources/reference.conf:124]: it looks like there's no need to enable the feature in every config file. Doing this in `reference.conf` should be enough

**Review by Vovchyk — CHANGES_REQUESTED** [review]

**Comment by rmoreliovlabs** [comment]
pipeline:run

**Review by Vovchyk — DISMISSED** [review]

**Review by Vovchyk — APPROVED** [review]

**Comment by Vovchyk** [comment]
pipeline:run

**Comment by Vovchyk** [comment]
pipeline:run

**Comment by Vovchyk** [comment]
pipeline:run

**Comment by Vovchyk** [comment]
pipeline:run

**Comment by rmoreliovlabs** [comment]
pipeline:run

**Comment by Vovchyk** [comment]
pipeline:run

**Comment by Vovchyk** [comment]
pipeline:run

**Pull request merged**